Gulpfile.mjs

```diff
@@ -393,8 +405,33 @@ function buildRollup(packages, targetBrowsers) {
           name,
           sourcemap: sourcemap,
           exports: "named",
+          interop(id) {
```

Comment:
The `interop` config fixes a regression issue: Using `@babel/plugin-proposal-optional-chaining` with an old `@babel/parser` version will throw optional chaining syntax is required.

In https://unpkg.com/@babel/plugin-proposal-optional-chaining@7.16.7/lib/index.js, 
```js
inherits: syntaxOptionalChaining__default['default'].default,
```
the inherited syntax plugin is always `undefined` because we and rollup both apply interop transform on the default exports.

Now it generates
```js
inherits: syntaxOptionalChaining.default,
```
as expected.